plugins/services/src/js/structs/__tests__/ServiceTree-test.js

```diff
@@ -1134,34 +1134,6 @@ describe("ServiceTree", () => {
       ]);
     });
 
-    it("filters out duplicate label key-value tuples", () => {
```

Comment:
we don't seem rely on this, so i did not bother resembling the original implementation exactly.